UserInterface/fileOutput/dummyPrinter_class.f90

```diff
@@ -23,6 +23,7 @@ module dummyPrinter_class
 
   contains
     procedure :: init
+    procedure :: endFile
     procedure :: extension
     procedure :: writeToFile
```

Comment:
I thought writeToFile was gone in the asciiOutput class?

Reply:
It is... I think this is an oversight that this got left in.

Reply:
Should be removed now.